README.md

```diff
@@ -1,6 +1,8 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
+![skopeo logo](https://cdn.rawgit.com/runcom/skopeo-1/add-logo/logo/skopeo.svg)
```

Comment:
(Pedantic reminder: this will probably need to point to the project atomic repo in the final version.)

Reply:
Consider using a raw `<img>` with width/height, or a smaller image; having only the logo above the fold may be a bit much.

Reply:
:+1: yeah btw, I'll switch to master :)